cetloxblock/cetloxblock/cetloxblock.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+""" 
+This XBlock displays a CET LO item.
+The studio view enables the author to select the LO for the current xblock instance.
+"""
+
+import pkg_resources
+#import urllib
+from xblock.core import XBlock
+#from xblock.fields import String, Scope, ScopeBase, UserScope, BlockScope
+from xblock.fields import String, Scope
+from xblock.fragment import Fragment
+
+class CetLoXBlock(XBlock):
+  """ 
+  This XBlock displays a CET LO item.
+  The studio view enables the author to select the LO for the current xblock instance.
+  """
+
+  CET_DOMAIN = u".lab.cet.ac.il"
+  ITEM_URL_TEMPLATE = u"//lo{cetdomain}/player/?document={documentid}&language={language}&sitekey=ebag#options=nobar"
+  TASK_URL_TEMPLATE = u"//lo{cetdomain}/player/xblockplayer.html?task={taskid}&sitekey=ebag"
+  #TASK_URL_TEMPLATE = u"//lo{cetdomain}/player/?task={taskid}&sitekey=ebag#options=nobar"

```

Comment:
In the official version of this XBlock (I suppose this PR is a part of making it official and production-ready), we should really keep all commented-out code out of the repository.